### DISCUSSION TITLE
CleanMyMacApp Moves From Section to Section

### DISCUSSION THREAD

**Discussion by jjulius50** [jordanbaird/Ice#187] [Q&A] [answered]
My preference is to keep the CleanMyMac (CMC) app in the Hidden Section. I have no problem moving the app there, but from time to time, the CMC app decides to locate itself in the Open Section or in the Always-Hidden Section. Is anyone else experiencing this and/or can anyone suggest a solution? Thanks

**Comment by jordanbaird** [comment, marked as answer]
This is definitely a known problem. In the future, Ice will have a profiles feature that will enable it to observe the location of each menu bar item and ensure they stay in the correct order.

**Comment by jjulius50** [comment]
Jordan, thank you for taking the time to personally respond. I know you must have a rather full plate.

I look forward to the new feature. This minor inconvenience will not cause me to go back to Bartender and/or look for different launcher.

Thanks again for the product and your time.

Jeff
Jeffrey Julius
***@***.***

Notice: This communication is intended only for the use of the individual or entity to which it is addressed and may contain information that is privileged, confidential or exempt from disclosure. If you have received it in error, please advise the sender by reply email and immediately delete the message and any attachments without copying and/or disclosing the contents. Thank you.
